components/schema-migrator/validate.sh

```diff
@@ -44,32 +45,48 @@ docker run -d --name ${POSTGRES_CONTAINER} \
             -e POSTGRES_DB=${DB_NAME} \
             postgres:${POSTGRES_VERSION}
 
-echo -e "${GREEN}Run UP migrations ${NC}"
-
-docker run --rm --network=${NETWORK} \
-        -e DB_USER=${DB_USER} \
-        -e DB_PASSWORD=${DB_PWD} \
-        -e DB_HOST=${POSTGRES_CONTAINER} \
-        -e DB_PORT=${DB_PORT} \
-        -e DB_NAME=${DB_NAME} \
-        -e DB_SSL=${DB_SSL_PARAM} \
-        -e DIRECTION="up" \
-    ${IMG_NAME}
-
-echo -e "${GREEN}Show schema_migrations table after UP migrations${NC}"
-docker exec ${POSTGRES_CONTAINER} psql -U usr compass -c "select * from schema_migrations"
-
-echo -e "${GREEN}Run DOWN migrations ${NC}"
-docker run --rm --network=${NETWORK} \
-        -e DB_USER=${DB_USER} \
-        -e DB_PASSWORD=${DB_PWD} \
-        -e DB_HOST=${POSTGRES_CONTAINER} \
-        -e DB_PORT=${DB_PORT} \
-        -e DB_NAME=${DB_NAME} \
-        -e DB_SSL=${DB_SSL_PARAM} \
-        -e DIRECTION="down" \
-        -e NON_INTERACTIVE="true" \
-    ${IMG_NAME}
-
-echo -e "${GREEN}Show schema_migrations table after DOWN migrations${NC}"
-docker exec ${POSTGRES_CONTAINER} psql -U usr compass -c "select * from schema_migrations"
+function migrationUP() {
+    echo -e "${GREEN}Run UP migrations ${NC}"
+
+    migration_path=$1
+    docker run --rm --network=${NETWORK} \
+            -e DB_USER=${DB_USER} \
+            -e DB_PASSWORD=${DB_PWD} \
+            -e DB_HOST=${POSTGRES_CONTAINER} \
+            -e DB_PORT=${DB_PORT} \
+            -e DB_NAME=${DB_NAME} \
+            -e DB_SSL=${DB_SSL_PARAM} \
+            -e MIGRATION_PATH=${migration_path} \
+            -e DIRECTION="up" \
+        ${IMG_NAME}
+
+    echo -e "${GREEN}Show schema_migrations table after UP migrations${NC}"
+    docker exec ${POSTGRES_CONTAINER} psql -U usr compass -c "select * from schema_migrations"
+}
+
+function migrationDOWN() {
+    echo -e "${GREEN}Run DOWN migrations ${NC}"
+
+    migration_path=$1
+    docker run --rm --network=${NETWORK} \
+            -e DB_USER=${DB_USER} \
+            -e DB_PASSWORD=${DB_PWD} \
+            -e DB_HOST=${POSTGRES_CONTAINER} \
+            -e DB_PORT=${DB_PORT} \
+            -e DB_NAME=${DB_NAME} \
+            -e DB_SSL=${DB_SSL_PARAM} \
+            -e MIGRATION_PATH=${migration_path} \
+            -e DIRECTION="down" \
+            -e NON_INTERACTIVE="true" \
+        ${IMG_NAME}
+
+    echo -e "${GREEN}Show schema_migrations table after DOWN migrations${NC}"
+    docker exec ${POSTGRES_CONTAINER} psql -U usr compass -c "select * from schema_migrations"
+}
+
+for i in "${MIGRATION_PATHS[@]}"
+do
+   echo -e "${GREEN}Migrations for \"${i}\" path${NC}"
```

Comment:
you wrote to the same database and to the same schema_migrations table - keep it separate.

Reply:
I used `Initialization scripts` ([link](https://hub.docker.com/_/postgres)) to create databases